buildenv/jenkins/JenkinsfileBase

```diff
@@ -773,7 +777,21 @@ def addGrinderLink() {
 def run_parallel_tests() {
 	if (params.PARALLEL && params.PARALLEL != "None") {
 		stage ("Parallel Tests") {
-			parallel parallel_tests
+			flow = parallel parallel_tests
```

Comment:
- Please use `def` for local variable
- Rename `flow ` to a meaningful name. Maybe `childJobs`